lib/membrane/rtp/packet.ex

```diff
@@ -73,28 +78,36 @@ defmodule Membrane.RTP.Packet do
         payload_type: payload_type,
         timestamp: timestamp,
         csrcs: for(<<csrc::32 <- csrcs>>, do: csrc),
-        extension: extension
+        extension: extension,
+        has_padding?: has_padding == 1,
+        total_header_size: 32 + 32 + 32 + 32 * csrcs_cnt + header_extension_size
```

Comment:
Agree about keeping the size in bytes. I would keep the size in either case when the header gets cut off the payload or not, it can be misleading and I can see a use case why we would event want to place `0` there.

Reply:
I'm wondering if it would be a good approach to keep both `total_header_size` and `has_padding` out of the RTP header and put that in buffer's metadata one level down (on the same level as `:rtp` or something like that? The header would then just keep properties that appears only in RFC and other elements if they need would access the properties separately.

Reply:
that's what I wanted to say 👍

Reply:
Done:)